02B-Python-Omop-walkthrough-critical-care/02B-Python-Omop-walkthrough-critical-care.md

```diff
@@ -0,0 +1,2006 @@
+This document is an introductory walkthrough, demonstrating how to read into Python some OMOP data from a series of csv files, join on concept names and explore and visualise the data.
+
+It uses synthetic data created in UCLH as a part of the  [Critical Care Health Informatics Collaborative (CCHIC) project](https://safehr-data.github.io/uclh-research-discovery/projects/uclh_cchic_s0/index.html).
+
+# Installing & loading required packages
+1. ### pip install pandas
+
+2. ### pip install seaborn
+
+3. ### pip install matplotlib
+
+
+
+```python
+import os
+import requests
+import pandas as pd
+import datetime
+import seaborn as sns
+import matplotlib.pyplot as plt
+```
+
+# Downloading & reading in OMOP data
+Here we will download & read in some UCLH critical care data stored in Github
+
+
+```python
+
+
+# Repository and file paths
+repository = "SAFEHR-data/uclh-research-discovery"
+source_path = "_projects/uclh_cchic_s0/data"
+destination_folder = "Starter_Guide_python/data"
```

Comment:
Can we also move the ipyn file to /dynamic-docs/02b-python-omop-walkthrough-critical-care.ipyn please 🙏🏻 

Then for this I think you want something like this (will need to add pathlib import above) so that we can download the data relative to the notebook to the same place as the R script


```suggestion
destination_folder = Path(__file__).parent / "data"
```